backend/src/main/java/ba/unsa/etf/si/bbqms/ws/models/AddTenantDto.java

```diff
@@ -0,0 +1,43 @@
+package ba.unsa.etf.si.bbqms.ws.models;
+
+public class AddTenantDto {
+    private final String code;
+    private final String name;
+    private final String hq_address;
```

Comment:
Java standard is `hqAddress`. [Reference](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)

Reply:
Changed to `hqAddress`